### PR TITLE
Update carousel_content.json to remove broken link

### PR DESCRIPTION
**Pull request by agurvich**
this carousel content was placeholder that should've been removed. The youtube link is not real, it would ideally be a collage of three images (I think I had composited them with the thumbnail).

In the future, it could be good to display a thumbnail instead of a youtube video if one is present in the carousel-content.json.

### CODE REVIEW

**Comment by agurvich** [comment]
I've gotten a separate request to include images in the carousel component as well. I think by passing along an extra key in the carousel-content metadata (`"format":"youtube"` or `"format":"image") we can conditionally use the `Embed` or `Image` mdx block. What do you think @hanbyul-here

**Comment by agurvich** [comment]
After explaining the lift to implement images (with thumbnails underneath) in the carousel we got a compromise proposal to break out the carousel: 

SpaceForEarth-SiteMockup2

All we would need for this is a color block background. Can we make that happen?

**Comment by hanbyul-here** [comment]
👋 
I believe the inline style like below should work 

```
<Block style={{backgroundColor: 'red'}}>
<Prose>
...
```

**Comment by agurvich** [comment]
Aha! I had no idea that styling was exposed there. This gives me so much more flexibility, thanks so much @hanbyul-here. You're my hero!

**Comment by hanbyul-here** [comment]
- Images are way too big, can you make sure if the images are not too big, and under 1mb? (This is the general guide for images: https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/media.md#media)

File: stories/locfeature.IMMER.mdx [hunk 1]
```diff
@@ -25,10 +25,181 @@ taxonomy:
 ---
 
 import Carousel from "../overrides/common/embedded-video-carousel";
-import contentArray from './locfeature.IMMER/carousel_content.json';
+import contentArray from './locfeature.IMMER/carousel_content';
```
Comment (hanbyul-here) [inline stories/locfeature.IMMER.mdx:28]: `Carousel ` component doesn't seem to be necessary anymore?

Comment (agurvich) [reply stories/locfeature.IMMER.mdx]: Yes, good call. I removed it from the .mdx file.

File: stories/locfeature.IMMER.mdx [hunk 2]
```diff
@@ -24,11 +24,179 @@ taxonomy:
...
 
-import Carousel from "../overrides/common/embedded-video-carousel";
-import contentArray from './locfeature.IMMER/carousel_content.json';
+import CardGallery from "./components/card_gallery";
+import { visitLocationFeaturesStoryIds } from '../overrides/common/story-data';
 
-<Block type='wide'>
+<Block >
+    <Prose>
+      ## Visit Space for Earth
```
Comment (hanbyul-here) [inline stories/locfeature.IMMER.mdx:32]: One nitpick: can you review the use of headings so they have a hierarchy? (https://www.w3.org/WAI/tutorials/page-structure/headings/) This page only has h2 and h4 now.

Comment (agurvich) [reply stories/locfeature.IMMER.mdx:32]: just saw this, done!

**Review by hanbyul-here — APPROVED** [review]
The contents layout looks nice!

**Pull request merged**